src/instance/mod.rs

```diff
@@ -18,27 +18,39 @@ use data::{Map,Monster};
 
 pub mod management;
 
+// Computation of the duration of one so-called 'tick'. A 'tick' is a
+// unite in the discrete time of lycan. By default, its value is set
+// to about 16ms.
```

Comment:
Indeed it could help spot tick problems earlier.

However the tick will basically affect the *precision* of Lycan, having a tick too long would probably make things a bit weird ... but we need to test.

Also, the tick value should not impact clients at all. It should be possible to send a number of updates (for example position of every player) depending on the bandwidth/power of the client, and thus the tick should not be tied to the client FPS.

So I will do a few tests with 5Hz, see how it goes.